piglow/piglow.go

```diff
@@ -0,0 +1,211 @@
+	if err := p.conn.Write([]byte{0x04, byte(level)}); err != nil {
+		return err
+	}
+
+	if err := p.conn.Write([]byte{0x06, byte(level)}); err != nil {
+		return err
+	}
+
+	if err := p.conn.Write([]byte{0x0E, byte(level)}); err != nil {
+		return err
+	}
+
+	if err := p.conn.Write([]byte{0x16, 0xFF}); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// Blue sets all the blue LEDs to the level of 0-255.
+func (p *PiGlow) Blue(level int) error {
+	if err := p.conn.Write([]byte{0x05, byte(level)}); err != nil {
+		return err
+	}
+
+	if err := p.conn.Write([]byte{0x0C, byte(level)}); err != nil {
+		return err
+	}
+
+	if err := p.conn.Write([]byte{0x0F, byte(level)}); err != nil {
+		return err
+	}
+
+	if err := p.conn.Write([]byte{0x16, 0xFF}); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// Yellow sets all the yellow LEDs to the level of 0-255.
+func (p *PiGlow) Yellow(level int) error {
+	if err := p.conn.Write([]byte{0x03, byte(level)}); err != nil {
+		return err
+	}
+
+	if err := p.conn.Write([]byte{0x09, byte(level)}); err != nil {
+		return err
+	}
+
+	if err := p.conn.Write([]byte{0x10, byte(level)}); err != nil {
+		return err
+	}
+
+	if err := p.conn.Write([]byte{0x16, 0xFF}); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// Orange sets all the orange LEDs to the level of 0-255.
+func (p *PiGlow) Orange(level int) error {
+	if err := p.conn.Write([]byte{0x02, byte(level)}); err != nil {
+		return err
+	}
+
+	if err := p.conn.Write([]byte{0x08, byte(level)}); err != nil {
+		return err
+	}
+
+	if err := p.conn.Write([]byte{0x11, byte(level)}); err != nil {
+		return err
+	}
+
+	if err := p.conn.Write([]byte{0x16, 0xFF}); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// White sets all the white LEDs to the level of 0-255.
+func (p *PiGlow) White(level int) error {
+	if err := p.conn.Write([]byte{0x0A, byte(level)}); err != nil {
+		return err
+	}
+
+	if err := p.conn.Write([]byte{0x0B, byte(level)}); err != nil {
+		return err
+	}
+
+	if err := p.conn.Write([]byte{0x0D, byte(level)}); err != nil {
+		return err
+	}
+
+	if err := p.conn.Write([]byte{0x16, 0xFF}); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// Red sets all the red LEDs to the level of 0-255.
+func (p *PiGlow) Red(level int) error {
+	if err := p.conn.Write([]byte{0x01, byte(level)}); err != nil {
+		return err
+	}
+
+	if err := p.conn.Write([]byte{0x07, byte(level)}); err != nil {
+		return err
+	}
+
+	if err := p.conn.Write([]byte{0x12, byte(level)}); err != nil {
+		return err
+	}
+
+	if err := p.conn.Write([]byte{0x16, 0xFF}); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// LED sets the led 1-18 to the level 0-255.
+func (p *PiGlow) LED(led, level int) error {
+	if err := p.conn.Write([]byte{byte(led), byte(level)}); err != nil {
+		return err
+	}
+
+	if err := p.conn.Write([]byte{0x16, 0xFF}); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// All sets all the LEDs to the level 0-255.
+func (p *PiGlow) All(level int) error {
```

Comment:
Should we call it SetBrightness? All is very generic to preserve for setting the level for all LEDs.